### PR TITLE
Update 2016-10-21-sweet-break-before-break-102.md

### PR DESCRIPTION
**Pull request by igotpassion**
Updates were made based on comments from @machikoyasuda and @renrutnnej. Thank you my friends!

### CODE REVIEW

**Comment by igotpassion** [comment]
@skeskali and @gr2m - I'm hoping I submitted changes using the correct workflow. Apologies if I did not. Let me know if you need anything else on my end for this.

File: _posts/2016-10-21-sweet-break-before-break-102.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday! This Friday’s lit
 
 Last week I hopped on a call with [Cecily](https://twitter.com/skeskali), [Gregor](https://twitter.com/gr2m), and [Jenn](https://twitter.com/jennwrites) to discuss the future of TGIF, Hoodie’s lazy Friday afternoon read started by [Lena](https://twitter.com/lrnrd) in [August 2013](http://hood.ie/blog/tgif-hoodies-top-links-of-the-week-1.html). After an hour-long chat with these amazing peeps we decided to take a step back and rethink how Hoodie’s weekly newsletter is created, delivered and promoted.
```
Comment (jennwrites) [inline _posts/2016-10-21-sweet-break-before-break-102.md]: I would update the timing on this since it’s been quite awhile since we all met.

**Comment by jennwrites** [comment]
I’m curious why there’s not a single pull quote from any of the listed reads, since that’s been a mainstay of the feature since the beginning.

Also what’s the significance in this format change as well, where the name of the site appears in parenthesis but isn’t linked?  

screen shot 2016-11-04 at 1 58 31 pm

File: _posts/2016-10-21-sweet-break-before-break-102.md (hunk 1 above)
Comment (igotpassion) [reply _posts/2016-10-21-sweet-break-before-break-102.md]: Nice catch.

**Comment by igotpassion** [comment]
@renrutnnej Good point on the pull quotes. I'll include some. I also decided to add sources in parentheses simply so people know what the source is. Some people might appreciate that. However only an A/B test could tell. (Maybe something we can do later.) I don't mind removing them for now if you think they are superfluous.

**Comment by igotpassion** [comment]
Looks like we'll have to push back another week. I'll submit another PR with updates based on Jenn's comments and some updated links.

**Comment by skeskali** [comment]
I like the addition of the sources for links. I also like pull quotes, but sometimes I felt they made TGIF a little too long. They were also the feature that made compiling TGIF the most time consuming, as you have to read every article to find the best pull quotes.

**Comment by igotpassion** [comment]
Thanks for your input Cecily! I'm going to include the sources this time around but do want to A/B test it in the future. I'm very interested if the inclusion of them has any impact on clickthroughs. As for the pull quotes, they do add a nice touch. I'll go ahead and add two of them.

**Comment by igotpassion** [comment]
@renrutnnej Also, just noticed that you weren't questioning the inclusion of the source necessarily but why it's not linked. I'll go ahead and link it as well. With 102 I wanted to test some different things out but don't want to interfere with the styleguide because it's solid.

**Comment by igotpassion** [comment]
Made changes. Let me know if y'all have any final suggestions or revisions. Thanks!

**Comment by skeskali** [comment]
Merged.

**Comment by igotpassion** [comment]
Is someone able to publish this today?

**Comment by gr2m** [comment]
I’m on it

**Comment by jdorfman** [comment]
great job @igotpassion <3

**Comment by igotpassion** [comment]
Thanks everyone!

**Pull request merged**